libsolidity/interface/CompilerStack.cpp

```diff
@@ -59,6 +59,8 @@ CompilerStack::CompilerStack(ReadFileCallback const& _readFile):
 void CompilerStack::setRemappings(vector<string> const& _remappings)
 {
 	vector<Remapping> remappings;
+	map<string, int> index;
+	map<string, int> contextReferences;
 	for (auto const& remapping: _remappings)
 	{
 		auto eq = find(remapping.begin(), remapping.end(), '=');
```

Comment:
so a directory name containing an `=` will be included so long as the first one is put in. However, perhaps this could be strengthened by making it a `find_first_of` rather than just a `find`.